config/main.py

```diff
@@ -5802,23 +5829,40 @@ def bind(ctx, interface_name, vrf_name):
         if interface_name is None:
             ctx.fail("'interface_name' is None!")
 
-    if not is_vrf_exists(config_db, vrf_name):
-        ctx.fail("VRF %s does not exist!"%(vrf_name))
+    isVnet = False
```

Comment:
I feel that its confusing if we are using a ```config vrf``` command to bind to a vnet object. Even though ```config vnet``` is what one would use to create a vnet, one must then issue a config vrf to bind an interface to a vnet.  I think its better to conform to a standard model, either we use all VRF or all VNET commands so that its not split between 2 types.